plugin.audio.navahang/default.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#  Copyright 2019 Amir Ammari
+#
+#  This file is part of the Radio Javan kodi plugin.
+#
+#  This plugin is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This plugin is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this plugin.  If not, see <http://www.gnu.org/licenses/>.
+
+from os.path import join
+from sys import argv
+import urllib2
+import json
+from time import gmtime, strftime, strptime
+from urllib import quote_plus, urlopen
+from urlparse import parse_qs, urlparse
+from xml.dom.minidom import parseString
+from xbmc import translatePath
+import xbmcaddon
+from xbmcgui import Dialog, ListItem
+from xbmcplugin import addDirectoryItem, endOfDirectory
+
+ADDON = xbmcaddon.Addon()
+ADDONNAME = ADDON.getAddonInfo('name')
+ADDONID = ADDON.getAddonInfo('id')
+ADDONVERSION = ADDON.getAddonInfo('version')
+CWD = ADDON.getAddonInfo('path').decode("utf-8")
+LANGUAGE = ADDON.getLocalizedString
+
+DataLink = 'http://navahang.me/webservice/GetMp3?type=Featured&from=0&to=20'

```

Comment:
```suggestion
DataLink = 'http://navahang.com/webservice/GetMp3?type=Featured&from=0&to=20'
```

Reply:
@Rechi I changed .me domain to official domain (.com domain)
Please check this plugin again and report problems.